arraySort.js

```diff
@@ -0,0 +1,31 @@
+var evenNumbers= new Array();
```

Comment:
- You were told to create a function that takes an array and returns an object. Can you do it as you were requested, please?